tests/filesTest.js

```diff
@@ -1171,16 +1285,20 @@ describe('Main: Currently testing files management,', function () {
 
     it('checking method: favorite filter', async function () {
       const provider = createProvider(true, true)
```

Comment:
should pass now in oc10, can you set it to `const provider = createProvider(false, true)`